package.json

```diff
@@ -12,6 +12,7 @@
   },
   "license": "MIT",
   "private": true,
+  "packageManager": "pnpm@8.3.1",
```

Comment:
missing `engines` with strict check for pnpm 8

Reply:
✅